ruby-service/app/services/database_query_handler.rb

```
@@ -0,0 +1,83 @@
+      CREATE TABLE IF NOT EXISTS code_reviews (
+        id INTEGER PRIMARY KEY AUTOINCREMENT,
+        user_id INTEGER NOT NULL,
+        content TEXT NOT NULL,
+        score REAL,
+        reviewed_at TEXT NOT NULL
+      );
+    SQL
+  end
+
+  def find_user_by_username(username)
+      @db.execute("SELECT * FROM users WHERE username = ? LIMIT 1", [username]).first
```

RuboCop (1.88.2)

[convention] Use 2 (not 4) spaces for indentation.

(Layout/IndentationWidth)